### ISSUE TITLE
is it possible to provide trust store public keys via ConfigMap?

### ISSUE THREAD

**Issue by WalkerWalker** [ibm-messaging/mq-helm#57]
After reading the readme, I found that it is only possible to provide trust store public keys via Secret

```
pki:
  trust:
    - name: default
      secret:
        secretName: appsecret
        items:
          - app.crt
```

Would it be possible to support providing trust via config map?  In our ocp cluster, the service ca is provided as a config map.

**Comment by callumpjackson** [comment]
Hi Walker - I've tagged this as a future enhancement request. For your awareness if you are using OpenShift then the strategic deployment option is the MQ Operator as documented [here](https://www.ibm.com/docs/en/ibm-mq/9.3?topic=openshift-operating-mq-using-mq-operator).

**Comment by WalkerWalker** [comment]
I can do a PR if it's welcomed.  Probably not too complicated.

**Comment by callumpjackson** [comment]
@WalkerWalker Thanks for the offer and normally I would welcome a PR. In this case I was in the middle of making a change and included within #60. This should now be live. Let me know if you experience any issues.
